### ISSUE TITLE
Native division does not throw ArithemticError

### ISSUE THREAD

**Issue by ndench** [pepakriz/phpstan-exception-rules#136]
In #77 checks for ArithmeticError were added, however I don't think that native division (eg. `$x / $y`) can actually throw this. https://github.com/pepakriz/phpstan-exception-rules/issues/77#issuecomment-477955886 mentions only two ways to get the error:

> 1.    intdiv(PHP_INT_MIN, -1);
> 2.    1 << -3;

And the [PHP docs for ArithmeticError](https://www.php.net/manual/en/class.arithmeticerror.php) don't mention anything about native division.

If it's not possible for it to be thrown here, maybe we shouldn't be reporting it as an issue?

**Comment by VincentLanglet** [comment]
What about https://www.php.net/manual/fr/class.divisionbyzeroerror.php
```
$x = 1;
$y = 0;
$x / $y;
```

**Comment by ndench** [comment]
@VincentLanglet extends ArithmeticError, so this code should be marked as throwing the more specific error. Though catching ArithmeticError at this point should silence any errors.

**Comment by VincentLanglet** [comment]
Maybe @pepakriz or @marcospassos have an example.

If not, do you want to make a PR @ndench ?

**Comment by pepakriz** [comment]
I haven't. So feel free to send PR

**Comment by marcospassos** [comment]
https://3v4l.org/Sv6lM
https://3v4l.org/kuD6g

**Comment by VincentLanglet** [comment]
> https://3v4l.org/Sv6lM
> https://3v4l.org/kuD6g

@marcospassos, I think that @ndench was asking for an example of `$x / $y` throwing an ArithmeticError which is not a `DivisionByZeroError`.
Because if there is no case, we could improve the possible exception thrown in this case.

This way
```
try {
  1 / 0;
} catch (DivisionByZeroError $error) {
  //
}
``` 
would not report an uncought ArithmeticError.
